test/modules/acbs/acbs-authentication.service.test.ts

```diff
@@ -0,0 +1,242 @@
+jest.mock('@nestjs/axios');
```

Comment:
I wonder if we need to mock axios at jest.config.ts level - it feels like this will be a global statement that we need for all tests

Reply:
Good idea, I'll add that